src/main/resources/org/jenkinsci/plugins/envinject/EnvInjectBuildWrapper/config.jelly

```diff
@@ -35,6 +21,20 @@
                 value="${instance.info.groovyScriptContent}"/>
     </f:entry>
 
+    <f:entry field="propertiesFilePath"
```

Comment:
@oleg-nenashev Indeed this change is not directly related to PR but I considered it would be helpful to show the form fields in the same order in both sections. Don't make sense to use a different order.